app/queries/User.ts

```diff
@@ -120,3 +121,86 @@ export const emailSubscribe = gql`
         }
     }
 `
+
+export const getBookmarkFolders = gql`
```

Comment:
Is it worth having a separate Bookmark file for the queries?

Reply:
done. see `queries/Bookmark.ts`